sdk/n8nsdk/api/openapi-source/v1/handlers/workflows/spec/schemas/workflowVersion.yml

```
@@ -0,0 +1,54 @@
+    readOnly: true
+    description: The workflow ID this version belongs to
+    example: 2tUt1wbLX592XDdX
+  nodes:
+    type: array
+    items:
+      $ref: './node.yml'
+    description: Nodes as they were in this version
+    readOnly: true
+  connections:
+    type: object
+    example: { Jira: { main: [[{ node: 'Jira', type: 'main', index: 0 }]] } }
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)